eureka/S5_lightcurve_fitting/fitters.py

```diff
@@ -11,10 +11,6 @@
 from .parameters import Parameters
 from .likelihood import computeRedChiSq, lnprob, ptform
 from . import plots_s5 as plots
-#FINDME: Keep reload statements for easy testing
```

Comment:
Keep reload statements for easy testing

Reply:
Did you see my issue at Issue #94 related to these reload statements?